user_test.py

```diff
@@ -311,7 +321,7 @@ def testRenderUserDetailsTemplate(self, mock_url_key):
     self.assertEquals('Invite Code Below' in user_details_template, False)
 
   @patch.object(user.User, 'key')
-  def testRenderUserDetailsTemplateWithInviteCode(self, mock_url_key):
+  def testRenderUserDetailsWithCode(self, mock_url_key):
```

Comment:
nit: I think we should keep the "Invite" in the method name.  It will much more clear.